samcli/commands/build/command.py

```diff
@@ -158,8 +158,19 @@ def do_cli(function_identifier,  # pylint: disable=too-many-locals
 
             click.secho("\nBuild Succeeded", fg="green")
 
-            msg = gen_success_msg(os.path.relpath(ctx.build_dir),
-                                  os.path.relpath(ctx.output_template_path),
+            # try to use relpath so the command is easier to understand, however,
+            # under Windows, when SAM and (build_dir or output_template_path) are
+            # on different drive, relpath() fails.
+            try:
+                build_dir = os.path.relpath(ctx.build_dir)
+                output_template_path = os.path.relpath(ctx.output_template_path)
+            except ValueError:
+                LOG.debug("Failed to retrieve relpath - using abspath instead")
+                build_dir = os.path.abspath(ctx.build_dir)
```

Comment:
Since we got a ValueError computing relative path, I would just use the original path without computing absolute path.